SETUP.md

```diff
@@ -0,0 +1,39 @@
+# Nylas Use Case Samples
+
+**⚠️ Warning: This repo is not meant for use in production and stability is not fully guaranteed.**
+
+Explore working front-end and back-end sample code using an integration with Nylas.
+
+The sample application redirects to a prebuilt authentication page hosted on Nylas, then allows that authenticated user to interact with Nylas API from a web client.
+
+## ⚙️ Environment Setup
+
+Let’s check that our environment is set up to use the [Nylas Node SDK](https://github.com/nylas/nylas-nodejs). Check the Node version in your terminal:
```

Comment:
Is this supposed to be Nylas Node SDK or Use Cases?

Reply:
oops, yeah this should be use cases. Good catch!